scripts/data-lint.js

```diff
@@ -0,0 +1,39 @@
+const inspect = require("util").inspect;
+
+const lib = require("./data-lint-lib");
+
+let errors = [];
+
+main();
+
+async function main() {
+	const { communitySchema, demosSchema, pluginsSchema, sitesSchema, startersSchema } = lib;
+
+	await Promise.all([
+		dataLint("../src/_data/community/", "*.js", communitySchema),
+		dataLint("../src/_data/demos/", "*.js", demosSchema),
+		dataLint("../src/_data/plugins/", "*.json", pluginsSchema),
+		dataLint("../src/_data/sites/", "*.json", sitesSchema),
+		dataLint("../src/_data/starters/", "*.json", startersSchema)
```

Comment:
This was interesting. I think it depends on how you run the script on whether this works.
I made some very obvious errors and it wasn't finding the templates or throwing errors, but I was running the script from the root directory as <kbd>node scripts/data-lint</kbd>, so I think it was looking in the wrong directory for the folders. I had to change these back to `./src/_data/community/`. But I guess it would work if we did a <kbd>cd scripts; node data-lint</kbd>.

Reply:
Either that or using `__dirname`.

Reply:
I had to install the dependencies to be able to run it.
Now you can execute it from the project root.